app/dashboard/ui.test.js

```diff
@@ -263,7 +271,7 @@ test('clicking on the client application type section link should navigate the u
 
 // TODO uncomment this test when contact-details page is complete
 // test('clicking on the contact details section link should navigate the user to contact details page', async (t) => {
-//   pageSetup(t);
+//   await pageSetup(t);
```

Comment:
I assume this test can now be active. Can be changed in another PR.

Reply:
ok